lib/moon/design/button/icon_button.ex

```diff
@@ -8,7 +8,7 @@ defmodule Moon.Design.Button.IconButton do
   alias Moon.Design.Loader
   alias Moon.Design.Button.Utils
 
-  @doc "Visual/Logical variant of button. Please use \"fill\", \"outline\", \"ghost\"  instead of \"primary\", \"secondary\", \"tertiary\" "
+  @doc "Visual/Logical variant of button."
```

Comment:
In addition, please also fix variant description for Moon.Design.Pagination.PrevButton and Moon.Design.Pagination.NextButton